generators/client/templates/react/package.json.ejs

```diff
@@ -41,7 +41,7 @@
     "react": "<%= this.dependabotPackageJson.dependencies['react'] %>",
     "react-dom": "<%= this.dependabotPackageJson.dependencies['react-dom'] %>",
     "react-hook-form": "<%= this.dependabotPackageJson.dependencies['react-hook-form'] %>",
-    "react-jhipster": "<%= this.dependabotPackageJson.dependencies['react-jhipster'] %>",
+    "react-jhipster": "jhipster/react-jhipster",
```

Comment:
change at package.json instead of the template file

Reply:
Ahhhh, thanks!